### PR TITLE
docs: Update instructions to run the validator

### PR DESCRIPTION
**Pull request by isabelle-dr**
**Summary:**
After receiving feedback from users on our instructions to run the validator via the command line, we identified the following pain points:
- the user didn't know that the file downloaded from `test-pack-doc` needed to be unzipped to access the jar file
- no mention that the user needs to navigate to the folder where the jar file is before running the command line (we can't use a path for the -jar argument)
- it's not clear what the country code argument is used for
- the fact that the command-line examples in USAGE.md aren't using the same parameter names as in README.md is confusing (the example in USAGE.md is using long names for parameters, and the example in README.md is using short names)

This PR solves these problems by:
- Updating the instructions in the README.md
- Adding information on what the country code is for
- Updating USAGE.md to have more homogeneity between the two

This PR also includes:
- an update to the description of InvalidPhoneNumberNotice in RULES.md to specify that this notice depends on the country code
- a fix in USAGE.md to switch Java 8 to Java 11
- added examples and fixed the requirements for the command line parameters in USAGE.md

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] ~~Run the unit tests with `gradle test` to make sure you didn't break anything~~
- [x] Format the title like "feat: [new feature short description]". Title must follow the Conventional Commit Specification(https://www.conventionalcommits.org/en/v1.0.0/).
- [x] Linked all relevant issues
- [x] Include screenshot(s) showing how this pull request works and fixes the issue(s)
## Instructions Before:
Screen Shot 2022-03-16 at 12 51 32 PM
## Instructions After:
Screen Shot 2022-03-16 at 12 51 12 PM
## Parameters Before:
Screen Shot 2022-03-16 at 12 53 47 PM

## Parameters After:
Screen Shot 2022-03-16 at 12 54 16 PM

### CODE REVIEW

File: docs/USAGE.md [hunk 1]
```diff
@@ -1,52 +1,51 @@
...
+| `-c`       | `--country_code`              | Optional               | Country code of the feed, e.g., `nl`. It must be a two-letter country code (ISO 3166-1 alpha-2). (e.g., `ca`, `us`). It can be either lower or upper case (e.g. `FR` or `GP`). If the country code is provided, phone numbers will be validated based on it.                             |
+| `-h`       | `--help`                      | Optional               | Print help menu.                                                                                                                                                                                                                                                                         |
+| `-t`       | `--threads`                   | Optional               | Number of java threads to use.                                                                                                                                                                                                                                                           |
```
Comment (maximearmstrong) [inline docs/USAGE.md:18]:
```suggestion
| `-t`       | `--threads`                   | Optional               | Number of threads to be used by Java to run the validator.                                                                                                                                                                                                                                                           |
```

File: docs/USAGE.md [hunk 2]
```diff
@@ -1,52 +1,51 @@
...
+| `-i`       | `--input`                     | Conditionally required | The path to the GTFS file (e.g., `/myDirectory/gtfs.zip`). Required if `-u` or `--url` is not provided.                                                                                                                                                                                  |
+| `-u`       | `--url`                       | Conditionally Required | `--url` or `-u`: the fully qualified URL to the GTFS file (e.g., `https://www.abc.com/gtfs.zip`). Required if `-i` or `--input` is not provided.                                                                                                                                         |
+| `-o`       | `--output`                    | Required               | Path where the validation report will be stored (e.g., `output`)                                                                                                                                                                                                                         |
```
Comment (maximearmstrong) [inline docs/USAGE.md:14]:
```suggestion
| `-o`       | `--output`                    | Required               | Path to where the validation report will be stored (e.g., `output`)                                                                                                                                                                                                                         |
```

**Review by maximearmstrong — CHANGES_REQUESTED** [review]
Great update! A few comments in-line before approval :)

**Comment by isabelle-dr** [comment]
Thanks for the feedback, I updated the instructions to replace [ with {. 
PTAL @maximearmstrong

**Review by maximearmstrong — APPROVED** [review]
LGTM! Thanks @isabelle-dr :)

**Pull request merged**